apps/reaper/test/unit/reaper/http/downloader_test.exs

```diff
@@ -32,6 +32,56 @@ defmodule Reaper.Http.DownloaderTest do
     assert response.url == "http://localhost:#{bypass.port}/file/to/download"
   end
 
```

Comment:
this downloader could stand to have more error case tests, but in general, that's not on you